### PR TITLE
Missena Bid Adapter : add session tracking, update referer & location fields

### PR DESCRIPTION
**Pull request by pdamoc**
## Type of change

- [x] Feature

## Description of change

This PR implements session tracking logic and restructures the referer related fields.

## Other information

@ysfbsf  is responsible for the implementation.

### CODE REVIEW

File: modules/missenaBidAdapter.js [hunk 1]
```diff
@@ -148,18 +148,19 @@ export const spec = {
...
       }
-      if (bidRequest.params.placement) {
-        payload.placement = bidRequest.params.placement;
+      if (bidParams.placement) {
+        payload.placement = bidParams.placement;
       }
-      if (bidRequest.params.formats) {
-        payload.formats = bidRequest.params.formats;
+      if (bidParams.formats) {
+        payload.formats = bidParams.formats;
```
Comment (patmmccann) [inline modules/missenaBidAdapter.js:160]: it seems you are just changing variable names to avoid the detection

Comment (ysfbsf) [reply modules/missenaBidAdapter.js:147]: How can we avoid triggering duplication errors in this case?

Comment (patmmccann) [reply modules/missenaBidAdapter.js:147]: You should import common functions from a library

**Comment by patmmccann** [comment]
It appears your commits have intended to fool the duplication check instead of fixing the underlying issue.

**Comment by patmmccann** [comment]
Also please note jsdoc issues: 


/home/circleci/Prebid.js/modules/missenaBidAdapter.js
   65:0  warning  Syntax error in namepath: -                                             jsdoc/valid-types
   65:0  warning  Expected @param names to be "validBidRequests, bidderRequest". Got "-"  jsdoc/check-param-names
  173:0  warning  The type 'data' is undefined                                            jsdoc/no-undefined-types
  173:0  warning  Expected @param names to be "timeoutData". Got "Containing"             jsdoc/check-param-names
  181:0  warning  Expected @param names to be "bid". Got "The"                            jsdoc/check-param-names

**Pull request closed without merging**